packages/renderer/src/lib/container/ContainerDetailsTerminal.spec.ts

```diff
@@ -286,8 +286,8 @@ test('prompt is not duplicated after restoring terminal from containerTerminals
   // terminal.write
   renderObject = render(ContainerDetailsTerminal, { container, screenReaderMode: true });
 
-  // wait shellInContainerMock is called
-  await waitFor(() => expect(shellInContainerMock).toHaveBeenCalled());
+  // wait shellInContainerMock is called a second time
+  await waitFor(() => expect(shellInContainerMock).toHaveBeenCalledTimes(2));
```

Comment:
question: maybe we are not suppose to call this function twice?

Reply:
There may be an optimization to do on the component side but currently it is called twice and the test is passing with this

Reply:
Okay we seems to unmount the component manually

https://github.com/containers/podman-desktop/blob/b60ecdc25ca2bc7d33432081e4bb6842ed80a712/packages/renderer/src/lib/container/ContainerDetailsTerminal.spec.ts#L282-L283

We may instead need to reset the mock, calling `vi.mocked(shellInContainerMock).mockClear();` after the unmount, as we seems to be confusing which call we are watiting for.

I would rather have a fix number, like `.haveBeenCalledOnce()`